src/DnnSummit.Data/Models/Location.cs

```diff
@@ -0,0 +1,34 @@
+﻿namespace DnnSummit.Data.Models
+{
+    public class Location
+    {
+        public string Title { get; set; }
+        public string Description { get; set; }
+        public string HotelName { get; set; }
+        public string HotelStandardsTitle { get; set; }
+        public string HotelStandard1 { get; set; }
```

Comment:
Could you update this public model to be an IEnumerable? I understand the schema from 2sxc is a mess due to some limitations of the data. If this is a big ask we can leave this as is.

Reply:
Just pushed a new commit to resolve.  Great idea.

Reply:
your changes look good,thanks!